docs/dev/setup.rst

```diff
@@ -273,7 +273,7 @@ You could use a compose file similar to this (as an example this deployment uses
 
     services:
       gitlab:
-        image: gitlab/gitlab-ce
+        image: ghcr.io/ls1intum/artemis-gitlab:latest
```

Comment:
I'm not sure whether the setup should refer to custom images of gitlab / jenkins
This would lead to maintenance effort for all depending images.

Maybe simply refer to a process on how to create the images on Mac